nih-sparc/sparc-app
# Fix download files and various Pennsieve migration - CU-tn5ka4

# Description

The purpose of this PR is to fix view and download files and various Pennsieve migration.

## Type of change


# How Has This Been Tested?

- [Go to a dataset](http://localhost:3000/datasets/86?type=dataset)
- Go to the Files tab
- Files should load
- Open the `primary` folder
- Scroll down to `PelotEtAl_2020_JNeurophys_Supplements.pdf` and click on the name
- The PDF should open in a new tab

# Checklist:
